### ISSUE TITLE
Broken Link on https://multibit.org/community.html

### ISSUE THREAD

**Issue by DuncanBetts** [bitcoin-solutions/multibit-website#274]
In the section regarding developers, the link to:
https://web.archive.org/web/20140319010125/http://bitcoindev.us.to/en/developer-guide
is dead (and has been for quite awhile).
Also everything's been deleted from the github repo (that's how it looks to me anyway):
https://github.com/saivann/bitcoin.org/blob/dev/en/developer-guide.md
I found a Archive.org Snapshot from 19 March 2014:
https://web.archive.org/web/20140319010125/http://bitcoindev.us.to/en/developer-guide)
I made a copy of this too: https://archive.is/Jpcqb

Hope that helps :)

**Comment by gary-rowe** [comment]
Thanks for the feedback - much appreciated.

The correct link for the developer guide is now https://bitcoin.org/en/developer-reference so I've updated the community page to reflect this. I don't think it's necessary to maintain the GitHub link any more.

I've also added a few more useful links to other sites that may benefit from extra traffic.

Finally, the FAQ has been updated to reflect 15+ million coins mined.

Ready for review and close.
